src/olympia/abuse/cinder.py

```diff
@@ -39,7 +39,9 @@ def get_relationship_data(self, to, relationship_type):
     def build_report_payload(self, *, report_text, category, reporter):
         context = self.get_context()
         if reporter:
-            context['entities'] += [reporter.get_entity_data()]
+            reporter_entity_data = reporter.get_entity_data()
+            if reporter_entity_data not in context['entities']:
+                context['entities'] += [reporter.get_entity_data()]
```

Comment:
something something `set`?

Reply:
I thought about using `set` but then I'd need to convert back into a `list` before passing it down to Cinder...